bbl_screen-patch/patches/printerui/qml/x1plus/Settings.js

```diff
@@ -63,7 +63,7 @@ function awaken() {
             _settingChanged(k, arg[k]);
         }
     });
-    _settings = X1Plus.DBus.proxyFunction("x1plus.x1plusd", "/x1plus/settings", "x1plus.settings", "GetSettings")() || {};
+    _settings = X1Plus.DBus.proxyFunction("x1plus.x1plusd", "/x1plus/settings", "x1plus.settings", "GetSettings")();
```

Comment:
I think you may want to revert the change to this line because I don't see anything in `settings.py` that directly prevents `GetSettings` from returning null or undefined. When it does have a null or undefined value, I'm fairly sure that results in a JS type error on line 69.

Reply:
I didn't actually edit this line. I haven't yet touched any of the Javascript files yet (that's coming in the next PR, when I add some Polar Cloud interface files), so I'm not sure what's happened. Is it possible that was in X1Plus's `main` at some point, and when I synced the fork it got pulled into my code? Or maybe I forked before the `or` was added in and git got confused on some merge or rebase?

The only files I changed were `.gitignore`, `polarcloud.py`, `__init__.py`, `utils.py`, and adding the Python libs.

In any case, yeah, I'll revert it.

Reply:
I figured that may be the case. I don't know what specifically causes this, but it seems to happen when rebasing a branch. I've had this happen too with code I have never touched, especially with patch files. [This](https://github.com/X1Plus/X1Plus/pull/290/files#diff-3f631b44f07bfba7f0ec409ed4c31ab27b6b55baaf66d41139b102e16ee426c3L81) looks like it might be another instance of this issue.

Reply:
Yeah, I did rebase at some point, which I rarely do, so I wasn't watching for this sort of thing. TIL.

I added those settings back.